src/modules/displayPop.js

```diff
@@ -0,0 +1,40 @@
+import productCount from './productCount.js';
+
+export default async function displayPop(data, comments) {
+  const moves = data.moves['0'].move.name;
+  const { weight } = data;
+  const { height } = data;
+  const ability = data.abilities[0].ability.name;
+  const { name } = data;
+  const { id } = data;
+  const img = data.sprites.other['official-artwork'].front_default;
+  const popUp = document.querySelector('.pop-up');
+  // const comment = await getpostInvolveCommentApi(id);
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

Reply:
Thank you for your review. We have changed it.